assets/css/busway_v2.scss

```diff
@@ -19,10 +19,10 @@
 @import "v2/lcd_common/no_data";
 @import "v2/lcd_common/page_load_no_data";
 
-@import "v2/busway/departures";
 @import "v2/lcd_common/departures";
-@import "v2/busway/departures_no_data";
 @import "v2/lcd_common/departures_no_data";
+@import "v2/busway/departures";
+@import "v2/busway/departures_no_data";
```

Comment:
**note:** I reordered these so that busway rules would override conflicting lcd_common rules.

Reply:
I think I'd prefer to set things up such that we minimize "overrides" — if a style differs between bus shelter and busway, then the things that are different should be in their respective screen-type-specific stylesheets, and there should not be any "default" in `lcd_common` that gets overridden. That said, if this were true, it wouldn't matter what order we import things in, so I wouldn't have any objection to this anyway.

Reply:
I'm in a bit of a pickle with this then. Let's chat and figure it out!

<details>
<summary>🥒</summary>

| When removing order change | Padding | CSS |
| - | - | - |
| <img width="587" alt="Screenshot 2024-05-24 at 9 25 33 AM" src="https://github.com/mbta/screens/assets/1699281/d96cad38-c37d-40d9-9e8f-90dbce863d55"> | <img width="606" alt="Screenshot 2024-05-24 at 9 25 48 AM" src="https://github.com/mbta/screens/assets/1699281/b004eb3e-2a19-47c0-8513-ae2aa05bc1df"> | <img width="500" alt="Screenshot 2024-05-24 at 9 26 09 AM" src="https://github.com/mbta/screens/assets/1699281/5fba014b-04c1-4308-b120-3a4eefe012ee"> |

</details>

Reply:
Summary of huddle convo: Let's move the top-padding that's currently shared in `lcd_common` to only apply to `bus_shelter`. Then, when a busway screen (only) has a first section with no visual header, the header component adds this padding (instead of its border). This way, when there _is_ a visual header, it's flush with the main screen header, and when there isn't, the padding is the same as with the bus shelter design.

Reply:
7b748aa2 all set!

| First empty header | Other empty header |
| - | - |
| <img width="1194" alt="Screenshot 2024-05-24 at 12 10 01 PM" src="https://github.com/mbta/screens/assets/1699281/1cd1c500-1ec8-4a3f-a7bd-d3c332ceb5d3"> | <img width="1260" alt="Screenshot 2024-05-24 at 12 10 12 PM" src="https://github.com/mbta/screens/assets/1699281/16547005-66dc-4a63-922b-fda6b283e1e8"> |